cmd/garm/main.go

```diff
@@ -166,6 +168,28 @@ func main() {
 		}
 	}()
 
+	if !cfg.APIServer.MetricsConfig.Disabled {
+		go func() {
+
+			metricsMiddleware := auth.NewMetricsMiddleware(cfg.JWTAuth)
+
+			r := mux.NewRouter()
```

Comment:
Please use the router inside the ```apiserver/router``` package. You can create a new sub-router from the main router. That router has only the logging middleware attached.

Reply:
done .. but I had to pass the config into the apirouter to allow enabling the metrics and the need for auth